### ISSUE TITLE
Difference to find beta 

### ISSUE THREAD

**Issue by aazz7777** [WillianFuks/tfcausalimpact#91]
I want to see the beta value and found tf.reduce_mean(ci.model.components_by_name['SparseLinearRegression/'].params_to_weights(
    ci.model_samples['SparseLinearRegression/_global_scale_variance'],
    ci.model_samples['SparseLinearRegression/_global_scale_noncentered'],
    ci.model_samples['SparseLinearRegression/_local_scale_variances'],
    ci.model_samples['SparseLinearRegression/_local_scales_noncentered'],
    ci.model_samples['SparseLinearRegression/_weights_noncentered'],
), axis=0) 

I compare with R pacakge. lots of beta is 0 but in here none of them are 0, I'm just wondering about the difference why none of beta does not show zero. For the output, if there a way to find coefficient for each iteration?
